Jenkinsfile

```diff
@@ -102,6 +102,10 @@ try {
         }
         parallel parallel_containers
 
+        if (env.BRANCH_NAME == 'master') {
+          build job: 'test-shiny-server-os', wait: false
+        }
+
         if (env.BRANCH_NAME == 'master') {
```

Comment:
I think you could also just add your `build` line to this `if` statement since they trigger identically.

Reply:
wanted to keep it separate, but no good reason :)